### ISSUE TITLE
False positive when return annotation is a string

### ISSUE THREAD

**Issue by Laleee** [jshwi/docsig#44]
Hello,

Here's a minimal example:

```python3
def example(some_input: int) -> int:
    """
    Do something.

    Args:
        some_input: Random integer

    Returns:
        Unchanged input
    """
    return some_input
```

This works like a charm but, when the return type is 'int' following error is returned. Sometimes, the return type has to be represented as a string due to dependency issues.

```
def example(✓some_input) -> ✖None:
    """
    :param some_input: ✓
    :return: ✖
    """
```
E104: return statement documented for None

**Comment by jshwi** [comment]
G'day @Laleee, thank you for pointing this out! I'll have a look at this, shouldn't be a problem 😄

**Comment by jshwi** [comment]
Should be fixed now in v0.30.1 :)
